domain/src/main/java/org/apache/rocketmq/eventbridge/domain/model/connection/EventConnectionWithBLOBs.java

```diff
@@ -0,0 +1,115 @@
+package org.apache.rocketmq.eventbridge.domain.model.connection;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.io.Serializable;
+
+public class EventConnectionWithBLOBs extends EventConnection implements Serializable {
+    /**
+     * This field was generated by MyBatis Generator.
+     * This field corresponds to the database column event_connection.auth_parameters
+     *
+     * @mbggenerated
+     */
+    private String authParameters;
```

Comment:
authParameters and networkParameters should be a struct model, and contains the type key.

Reply:
This class operates the database, so the object is finally converted into JSON and stored in these fields.